src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
```

Comment:
Could naming the variable `by` into `dueBy`, or `dueDate` help make it more readable?